### PR TITLE
stolostron/backlog#18298 img & perm

### PR DESCRIPTION
**Pull request by cadawson**

### CODE REVIEW

File: clusters/cluster_lifecycle_architecture.adoc [hunk 1]
```diff
@@ -25,7 +25,35 @@ Components on the hub cluster:
...
++
+** Role created in the hub cluster namespace of the hub cluster
++
+*** Allows the Work Agent to send events to the hub cluster 
++
+*** Allows the agent to `get`/`list`/`watch`/`update` the `manifestworks` resource
++
+*** Allows the agent to update the status of `manifestworks` resource 
+
 * Klusterlet Add-on Operator: Deploys the add-on components.
```
Comment (qiujian16) [inline clusters/cluster_lifecycle_architecture.adoc:56]: this Klusterlet Add-on Operator is removed in 2.5

**Comment by qiujian16** [comment]
@cadawson the description looks fine. We need to remove klusterlet addon operator in the diagram

**Comment by cadawson** [comment]
Thank you @qiujian16 . In order to remove the Klusterlet Add-on Operator, I need to clarify what to do with the connections to it. Here is a guess at the required changes to the diagram: 
1. Klusterlet Add-on Operator box is removed.
2. The connection from Add-on Custom Resource to Klusterlet Add-on Operator now connects to Add-on Controller?
3. The connection line from the Work Agent to the Klusterlet Add-on Operator is removed completely?
4. The connection from Add-on Custom Resource now goes directly to Add-on Controller, rather than through the Klusterlet Add-on Operator?

Let me know if these are the correct assumptions, and I can move forward with getting the diagram updated. Thank you!

**Comment by qiujian16** [comment]
> * Klusterlet Add-on Operator box is removed.

yes

> * The connection from Add-on Custom Resource to Klusterlet Add-on Operator now connects to Add-on Controller?

Add-on Custom Resource is removed also

> * The connection line from the Work Agent to the Klusterlet Add-on Operator is removed completely?

yes

> * The connection from Add-on Custom Resource now goes directly to Add-on Controller, rather than through the Klusterlet Add-on Operator?

work agent connects to Add-on Controller

**Pull request merged**